### PR TITLE
chore: align with AA4

### PR DESCRIPTION
**Pull request by dvilelaf**
- Populate contributing
- Remove changelog
- Align with AA4

### CODE REVIEW

File: .github/workflows/main_workflow.yml [hunk 1]
```diff
@@ -183,7 +179,6 @@ jobs:
           tar -xf tendermint.tar.gz
           sudo mv tendermint /usr/local/bin/tendermint
           # pull node image needed for the integration tests
-          docker pull node:16.7.0
           docker pull trufflesuite/ganache:beta
```
Comment (DavidMinarsch) [inline .github/workflows/main_workflow.yml:182]: you need to pull the two other images (same mentioned on readme)

File: .github/workflows/main_workflow.yml [hunk 2]
```diff
@@ -236,7 +227,7 @@ jobs:
           sudo mv tendermint /usr/local/bin/tendermint
 
           # pull node image needed for the integration tests
-          docker pull node:16.7.0
+          docker pull trufflesuite/ganache:beta
```
Comment (DavidMinarsch) [inline .github/workflows/main_workflow.yml:232]: same

File: AUTHORS.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
 # Authors
 
-This is the official list of authors for copyright purposes.
\ No newline at end of file
+This is the official list of authors for copyright purposes.
+
+* David Minarsch <david.minarsch@googlemail.com> [DavidMinarsch](https://github.com/DavidMinarsch)
```
Comment (DavidMinarsch) [inline AUTHORS.md]: No, this is a template repo, remove myself

File: .github/workflows/main_workflow.yml (hunk 1 above)
Comment (dvilelaf) [reply .github/workflows/main_workflow.yml:182]: Oh, I didnt include them as I thought for a template we wouldn't be including that. Fixing now.

File: README.md [hunk 4]
```diff
@@ -6,20 +6,23 @@ A template for development with the open-autonomy framework. Find the documentat
 
 Read the requirements section [here](https://docs.autonolas.network/quick_start/).
 
-## It contains:
+## Simple ABCI example
```
Comment (DavidMinarsch) [inline README.md]: ? there's none

Comment (DavidMinarsch) [reply README.md]: Please have structure here with deps, pull images, env, pull packages (even though nothing will be pulled)

File: tox.ini [hunk 5]
```diff
@@ -24,11 +24,12 @@ deps =
     hypothesis==6.21.6
     py-eth-sig-utils==0.4.0
     aiohttp==3.7.4.post0
-    open-autonomy==0.2.1
-    open-aea[all]==1.17.0
-    open-aea-ledger-ethereum==1.17.0
-    open-aea-ledger-cosmos==1.17.0
-    open-aea-cli-ipfs==1.17.0
+    open-autonomy==0.2.2
```
Comment (DavidMinarsch) [inline tox.ini]: wrong versions

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 5 above)
Comment (dvilelaf) [reply tox.ini]: Ok, I copied the whole file from AA4 since there were a lot of changes. Updating to latest.

File: README.md (hunk 4 above)
Comment (dvilelaf) [reply README.md]: Ok, I added some more things, but I'm confused about the pull packages thing: do you mean adding some remote packages from IPFS?

File: README.md [hunk 6]
```diff
@@ -4,22 +4,46 @@ A template for development with the open-autonomy framework. Find the documentat
 
 ## System requirements
 
-Read the requirements section [here](https://docs.autonolas.network/quick_start/).
+- Python `>=3.7`
+- Yarn `>=1.22.xx`
+- Node `>=v12.xx`
```
Comment (DavidMinarsch) [inline README.md]: they are not needed anymore

File: README.md [hunk 7]
```diff
@@ -4,22 +4,46 @@ A template for development with the open-autonomy framework. Find the documentat
...
-## It contains:
+Alternatively, you can fetch this docker image with the relevant requirments satisfied:
+
+        docker pull valory/dev-template:latest
+        docker container run -it valory/dev-template:latest
+
+- Build the Hardhat projects:
+
+      cd third_party/safe-contracts && yarn install
+      cd ../..
```
Comment (DavidMinarsch) [inline README.md]: this is not relevant. They can pull the images you list in github workfliow

File: tox.ini (hunk 5 above)
Comment (DavidMinarsch) [reply tox.ini]: https://github.com/valory-xyz/agent-academy-4/blob/main/tox.ini then you are behind main

File: README.md [hunk 8]
```diff
@@ -4,22 +4,34 @@ A template for development with the open-autonomy framework. Find the documentat
 
 ## System requirements
 
-Read the requirements section [here](https://docs.autonolas.network/quick_start/).
+- Python `>=3.7`
+- [Tendermint](https://docs.tendermint.com/master/introduction/install.html) `==0.34.19`
+- [IPFS node](https://docs.ipfs.io/install/command-line/#official-distributions) `==0.6.0`
+- [Pipenv](https://pipenv.pypa.io/en/latest/install/) `>=2021.x.xx`
```
Comment (DavidMinarsch) [inline README.md:11]: Docker & Docker compose too

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (dvilelaf) [reply README.md:11]: I was also missing pip & gitleaks.

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**